org.mozilla.Firefox.appdata.xml

```diff
@@ -65,7 +62,11 @@
   </content_rating>
 
   <developer_name>The Mozilla Foundation</developer_name>
-  <url type="homepage">https://www.mozilla.org/</url>
-  <url type="bugtracker">https://bugzilla.mozilla.org/describecomponents.cgi?product=Firefox</url>
+
+  <url type="homepage">https://www.mozilla.org/firefox/</url>
+  <url type="donation">https://donate.mozilla.org/</url>
+  <url type="bugtracker">https://bugzilla.mozilla.org/</url>
+  <url type="help">https://support.mozilla.org/</url>
```

Comment:
This entry is duplicated below, I would probably use this new URL here though given that the old URL is en-US specific.